plugin/build_baron.go

```diff
@@ -36,7 +37,21 @@ func (bbp *BuildBaronPlugin) Configure(conf map[string]interface{}) error {
 	}
 
 	for projName, proj := range bbpOptions.Projects {
-		webhookConfigured := proj.TaskAnnotationSettings.FileTicketWebHook.Endpoint != ""
+		var webHook evergreen.WebHook
+		flags, err := evergreen.GetServiceFlags()
+		if err != nil {
+			return errors.Wrap(err, "error retrieving admin settings")
+		}
+		if flags.PluginAdminPageDisabled {
+			projectRef, err := model.FindOneProjectRef(projName)
```

Comment:
We're checking project ref; should we also be checking parser project if we're going to be allowing this to be defined in the yaml? (to do this we would probably get the parser project for the most recent version on this project)